### ISSUE TITLE
Pages: Not possible to edit or delete Posts Page

### ISSUE THREAD

**Issue by kathrynwp** [Automattic/wp-calypso#32533]
Recently, the ability to edit or delete the page set to display the latest posts was removed.

See: #32022 & #31917

This is problematic, because it means users can no longer change the page title, add a featured image, add content, or delete this page in Calypso, for example.

#### Steps to reproduce

1. Starting at URL: Any site running the Lodestar theme (or another theme that uses posts-page content like the featured image
2. At Customize > Homepage Settings, under Posts page > Add New Page, type Blog and click "Add" select it as the front page 
3. At Customize > Theme Options, select the Blog page under Panel 1
4. Click Publish to save Customizer changes
5. Decide you want to call the page News instead, and assign a featured image to it so it'll be pulled into and displayed on the front page panel
6. Under Site Pages > Blog, click the title "Blog" to edit it, or click the three vertical dots.

#### What I expected

I expected to be able to edit the page to add a featured image and change the title. 

#### What happened instead

I was unable to edit the page in any way. Instead, it just opened the page on the front end.

#### Other use cases

What if a user decides they no longer want to have a blog or news section? There's no way to delete the page without unassigning it as the posts page first. The user may not realize they have to unassign it first, so they'll be stuck.

What if a user wants to see "News" instead of "Blog" in the editor? There's no way to rename the page.

What if a user is using a theme that displays post-page content? (There are some.) There's no way to add the content.

#### Context / Source

This came up during a Business Concierge session with a WordPress.com user. We wanted to add a featured image to the blog page but were unable to do so from Calypso.

Workarounds would be to use WP Admin, or to unassign the page from posts (Customize > Homepage Settings), edit the page, and then go back into Homepage Settings and reassign it as the posts page, which is cumbersome, and also not intuitive, as most users won't realize why they're being blocked from editing the page. In fact, I (an experienced Happiness Engineer) couldn't figure it out until I asked some colleagues, one of whom found the recent GitHub PR.

It seems that in solving one issue, a whole bunch more have been caused.

**tl;dr: Users need to be able to edit the posts page in Calypso**

#user-report

**Comment by kathrynwp** [comment]
Reference:

site - https://seoulofawriter.com/
ticket: 1981023-zen

**Comment by kathrynwp** [comment]
This came up again in 1984548-zen

**Comment by xpurichan** [comment]
I experience this one in chat as well, but it was weeks ago and I don't recall the HC ID. We did perform the workaround of unassigning the page, changing the title (which was the only thing they wanted to do), and then reassigning it.

This can also be a concern when a user has added content directly to that page, assigned it, then they can't access that content again. It happened occasionally with the old setup and users would come in to chat wondering why the blog page didn't display their content. Fortunately we could help them visit the blog page, cut the content, then add it to a new page; now there is a little extra dance to do.

I definitely agree that the blog page setup should be changed (maybe something similar to how our category pages are setup via the menu?), I'm not sure if this current iteration is the best solution.

**Comment by lakellie** [comment]
I don't have the interaction IDs handy unfortunately, but I've encountered this twice so far: someone trying to edit the featured image on the blog page, and someone else trying to change the title, both users were very confused as to why they couldn't

**Comment by xpurichan** [comment]
Another report 12352651-hc.

**Comment by sarahcada** [comment]
Reported at #12991369-hc. User was able to fix it in WP Admin.

**Comment by nagpai** [comment]
I had to send a user to WP Admin here:
13346733-hc
Followup - 2100557-zd

**Comment by tbradsha** [comment]
User needed to change their permalink, so I had to send them to WP Admin: 13265639-hc

**Comment by erricgunawan** [comment]
Had a user ask how to edit their Posts page title: 14752733-hc

I gave them both workaround:

> to unassign the page from posts (Customize > Homepage Settings), edit the page, and then go back into Homepage Settings and reassign it as the posts page

> to use WP Admin

**Comment by eduardozulian** [comment]
2433028-zen as well.

Gave them the wp-admin workaround.

**Comment by spencerberry** [comment]
#16169950-hc 
Wanted to update their blog page permalink, had to send through WP Admin to avoid My Site/s > Design > Customize > Homepage Settings dance.

**Comment by eduardozulian** [comment]
2491945-zen

Wanted to delete their posts page.

**Comment by tbradsha** [comment]
19207741-hc

Wanted to update their page title so it had the right name when sharing.

**Comment by jordesign** [comment]
Flagging this for Manage Focus to consider. Is it worth enabling the editing of the Blog Posts page in order to change Title and Feature Image?

If it isn't - then lets mark this as won't-fix and confirm user's would need to use the workaround of editing in WP Admin.

**Comment by liviopv** [comment]
Another in 28003564-hc

The user needed to edit the Featured Image of a Posts Page because their theme (Ovation) displays it on the Front Page. They had to unset it as Posts Page, edit the page, and reset as Posts Page.

**Comment by tbradsha** [comment]
This is still an issue, but now WP.com's WP Admin seems to block edit/delete access as well.

**Comment by eduardozulian** [comment]
Still an issue: 37188910-hc

**Comment by mgozdis** [comment]
Another user report here 6316791-zd-woothemes on Simple site.
They wanted to rename blog page title. Only workarounds I found when testing were to "Set as Regular Page", rename, and then set back as posts page, or create an entirely new blog page. Definitely not a user-friendly experience. 

On Simple sites, wp-admin and Classic View is no longer a workaround and can not edit or delete. AT sites you can.
